seqr/views/apis/project_api_tests.py

```diff
@@ -90,6 +92,7 @@ def test_create_project_no_pm(self):
         self.assertSetEqual(set(response.json()['projectsByGuid'].keys()), {new_project.guid})
 
     @mock.patch('seqr.views.utils.permissions_utils.ANALYST_PROJECT_CATEGORY', 'analyst-projects')
+    @mock.patch('seqr.views.utils.orm_to_json_utils.ANALYST_USER_GROUP', 'analysts')
     @mock.patch('seqr.views.utils.permissions_utils.ANALYST_USER_GROUP')
```

Comment:
I mean like yeah if you mocked them to be different values who knows what that would do, but test are only supposed to test real behavior, and there is no way for the one variable to somehow have different values in different files. The fact that we have to mock it in every place its imported instead of just in settings.py is an artifact of how the mock library works, but these are referencing the same variable so in the real world they will always be the same.